pom.xml

```diff
@@ -9,7 +9,7 @@
 	<description>RDFRank plugin for GraphDB</description>
 
 	<properties>
-		<graphdb.version>10.0.0-RDF4J4-BOOTSTRAP1</graphdb.version>
+		<graphdb.version>10.0.0-M2-TR9</graphdb.version>
 
 		<java.level>1.8</java.level>
```

Comment:
I know this is not related to the MR, but should we change the java version here as well